eva-lib/src/main/java/uk/ac/ebi/eva/lib/utils/QueryUtils.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.lib.utils;
+
+import com.google.common.base.Splitter;
+import org.opencb.datastore.core.QueryOptions;
+import org.opencb.datastore.core.QueryResponse;
+import org.opencb.datastore.core.QueryResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class QueryUtils {
+
+    protected final String version = "v1";
+
+    @Autowired
+    protected HttpServletRequest httpServletRequest;
+
+    private QueryOptions queryOptions;
+    protected long startTime;
+    protected long endTime;
+
+    protected static Logger logger = LoggerFactory.getLogger(QueryUtils.class);
+
+    public QueryOptions getQueryOptions() {
```

Comment:
does this need to be public? what advantage does this method provide in contrast with a protected attribute? are the startTime and endTime required to be protected still?